### DISCUSSION TITLE
why pytest log has arrow symbol when out put log to file?

### DISCUSSION THREAD

**Discussion by YYYGH** [pytest-dev/pytest#7633] [General] [answered]
- This is my logs, I got it from the report directory xxx-attachment.txt file. Where do those arrow symbols come from?
```
[2020-08-10 17:01:59] conftest.py[line:69] [32mINFO[0m init account, account_info:[]
[2020-08-10 17:02:15] login.py[line:190] [32mINFO[0m set msg code, phone:[18688888888], msgcode:[411062]
[2020-08-10 17:02:20] login.py[line:183] [32mINFO[0m msg:[login mssp success], username:[]
```

- python version: 3.8.4
pytest==5.4.3
- system version: windows7
- log config:
```python
logging.basicConfig(level=loglevel,
         format='%(asctime)s %(filename)s[line:%(lineno)d] %(levelname)s %(message)s',
         datefmt='[%Y-%m-%d %H:%M:%S]' )
run command:
log_format='%(asctime)s %(filename)s[line:%(lineno)d] %(levelname)s %(message)s'
log_date_format='[%Y-%m-%d %H:%M:%S]'
args = ["--log-level", "info", "--log-format", log_format, "--log-date-format", log_date_format, 
            "--tb", "short", "--capture", "no", "--alluredir", REPORT_DIR, API_TEST_CASE_DIR]
test_result = pytest.main(args)
```

**Comment by Zac-HD** [comment, marked as answer]
Looks like a problem with ANSI escape codes for colored output.

Related links: https://en.wikipedia.org/wiki/ANSI_escape_code , https://no-color.org/ , https://docs.pytest.org/en/stable/reference.html#envvar-PY_COLORS

**Comment by nicoddemus** [comment]
@Zac-HD, I've moved a few questions which have been closed recently by you (thanks again for that!), which have a clear answer, to a Discussion; I think this is a good approach we can take when closing old questions. 👍
